src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```diff
@@ -175,6 +176,10 @@ public final class ReblockGVCF extends MultiVariantWalker {
     @Argument(fullName=KEEP_SITE_FILTERS_LONG_NAME, shortName = KEEP_SITE_FILTERS_SHORT_NAME, doc="Keep site level filters for variants (not ref blocks).")
     private boolean keepFilters = false;
 
+    @Advanced
+    @Argument(fullName= ADD_FILTERS_TO_GENOTYPE, doc="Add site level filters to genotype level. Site level filters removed by default, if they should be kept, use --" + KEEP_SITE_FILTERS_LONG_NAME)
```

Comment:
Do we already have a test for KEEP_SITE_FILTERS alone?

Reply:
Yeah, KEEP_SITE_FILTERS was put in earlier and has a test.